src/var.c

```diff
@@ -53,8 +55,37 @@ struct fi_var_t {
 
 static struct fi_var_t *fi_vars = NULL;
 
+__attribute__((visibility ("default")))
+int DEFAULT_SYMVER_PRE(fi_getvars)(struct fi_env **vars, int *count) {
+
+	struct fi_env *vhead;
+	int len = 0;
+
+	// just get a count
+	for (struct fi_var_t *ptr = fi_vars; ptr ;ptr = ptr->next, ++len) {}
```

Comment:
nitpicking:
Move {} to next line (and consider replacing with just ;
Just declare ptr variable at the top of the call, rather than declaring it twice
Also, leave a space after the ; but not before in the for loop